doc/22_Administration_of_Pimcore/07_Users_and_Roles.md

```diff
@@ -55,62 +63,48 @@ The following list outlines what the different system permissions (available for
 
-A user will be granted any system permission that is granted to them directly or to any role they incorporate. 
-A permission granted to a role incorporated by an individual user, can not be rescinded for that individual user. So it 
-does not matter if the checkbox in the user's individual permissions settings is unchecked once a permission is granted 
-through a role.
+Users will benefit from any system permission that is granted to them directly or to any role they incorporate. 
+Permission granted to a role incorporated by an individual user can not be rescinded for that particular user. So it does not matter if the user's individual permissions settings checkbox is unchecked once this permission is granted through a role.
 
 ### Element Permissions - Workspaces
 
-Beyond the permissions mentioned above, a user's access can be restricted on element basis. This can be done by defining 
-workspaces for a user or role. Provided that a user may generally access documents, it can be specified what a user / role 
-may do or not do with each document or workspace. The same is true for objects and assets. These settings are manipulated 
-in the "Workspaces" tab of a user / role. 
+Beyond the permissions mentioned above, a user's access can be restricted on element basis. This can be done by defining workspaces for a user or role. Provided that a user may generally access documents, it can be specified what a user/role may do or not do with each document or workspace. The same is true for Data Objects and Assets. These settings are manipulated in the `Workspaces` tab of a user/role. 
 
-A user needs to be granted access to one or more workspaces. The user can not access any resources outside of his workspace(s). 
+A user needs to be granted access to one or more workspaces. The user can not access any resources outside his workspace(s). 
 
-However, there are a few general rules on element permissions which need to be regarded:
-* if a user does not have the right to list an element, all other permissions are obsolete
-* if a user does not have the list permission on an element, all permissions on this element's children are obsolete
+However, there are a few general rules on element permissions that need to be regarded:
+* if an user does not have the right to list an element, all other permissions are obsolete.
+* if an user does not have the list permission on an element, all permissions on this element's children are obsolete.
 
 ![User permission workspaces](../img/permissions1.png)
 
 The user permissions on element basis are summed up as follows:
 
-* **list**: element can be listed in tree
-* **view**: element can be opened
-* **save**: element can be saved (save button visible)
-* **publish**: element can be published (publish button visible)
-* **unpublish**: element can be unpublished (unpublish button visible); does not exist for assets
+* **list**: elements can be listed in tree
```

Comment:
these permissions really are for that particular element ... so we should leave that with `element`

Reply:
I changed that :)